### ISSUE TITLE
If you close the properties dialog before a file info is displayed, sometimes the app crashes

### ISSUE THREAD

**Issue by xPoppyx** [files-community/files-uwp#697]
This happens with large files. sometimes it crashes immediately, other times a few seconds later.
it also takes too long to display the info of large files

![crash](https://user-images.githubusercontent.com/61259627/80283510-0e601780-86e6-11ea-901e-0392d5bfcbfa.gif)

**Comment by RomanKornev** [comment]
Can confirm it crashes
```
2020-05-27 22:37:50.1861|INFO|Files.App|App launched
2020-05-27 22:54:12.2361|ERROR|Files.App|Element not found. (Exception from HRESULT: 0x80070490)
```

**Comment by xPoppyx** [comment]
fixed: #706
